lib/pdk/validators/metadata_validator.rb

```diff
@@ -18,6 +18,11 @@ def self.metadata_validators
       def self.invoke(report, options = {})
         exit_code = 0
 
+        if options[:targets] && options[:targets] != []
+          PDK.logger.info(_('metadata validator only checks metadata.json. The specified files will be ignored: %{targets}') % { targets: options[:targets].join(', ') })
+          options.delete(:targets)
```

Comment:
I was going to change this to remove :targets only in a copy of options passed to the two metadata validators.
It might be better to `dup` the options in `cli/validate.rb` to allow for further future special modifications to the options; cheers @bmjen